pkg/manager/member/orphan_pods_cleaner.go

```diff
@@ -88,8 +87,8 @@ func (opc *orphanPodsCleaner) Clean(tc *v1alpha1.TidbCluster) (map[string]string
 			continue
 		}
 
-		if pod.Status.Phase != v1.PodPending {
-			skipReason[podName] = skipReasonOrphanPodsCleanerPodIsNotPending
+		if len(pod.Spec.NodeName) > 0 {
+			skipReason[podName] = skipReasonOrphanPodsCleanerPodHasBeenScheduled
```

Comment:
we should not rely on status information here because the pod may be scheduled but the status may not be updated by the kubelet yet. 

however, if a pod has not been scheduled, it's safe to delete it whether PVCs have been created for it or not.